### ISSUE TITLE
don't show widgets on array or code blocs

### ISSUE THREAD

**Issue by patriciogonzalezvivo** [tangrams/tangram-play#211]

**Comment by louh** [comment]
@patriciogonzalezvivo Do you remember what this was about?

**Comment by patriciogonzalezvivo** [comment]
Yes... prevent widgets to mess with arrays like `color: [[15,[0.,0,0]], [20,[1.,1.,1.]]]` or JS blocks like `color: function() {return [1.,0,0]}`

**Comment by louh** [comment]
Got it - do we have an idea of how we want to address this? Should a color picker mark never appear if the value string is not actually parseable by the color picker? This is similar to how Chrome's inspector works.

**Comment by bcamper** [comment]
In the first case the ideal would be to create a color picker for each of
the inner color values... But that obviously requires a lot more parsing.

For JS functions, makes sense to show nothing I think.

**Comment by louh** [comment]
What does it actually mean when a color value is `[[15,[0.,0,0]],
[20,[1.,1.,1.]]]`? Is this documented anywhere?

Also would it be equal to expressing this in YAML's sequence notation?

```yaml
color:
  -
    - 15
    -
      - 0.
      - 0
      - 0
  -
    - 20
    -
      - 1
      - 1
      - 1
```

**Comment by bcamper** [comment]
It means it has multiple values interpolated by zoom:

https://mapzen.com/documentation/tangram/yaml/#stops
